packages/jest-util/src/install_common_globals.js

```diff
@@ -33,5 +34,7 @@ export default function(globalObject: Global, globals: ConfigGlobals) {
   globalObject.setImmediate = global.setImmediate;
   globalObject.clearImmediate = global.clearImmediate;
 
+  addInstanceOfAlias(globalObject.Error, Error);
```

Comment:
We should probably have some sort of blacklist and just loop all globals, rather than patching them in as people report bugs.

Reply:
Yes, please.